WordPress/Classes/Models/ManagedAccountSettings+CoreDataProperties.swift

```diff
@@ -0,0 +1,17 @@
+import Foundation
+import CoreData
+
+extension ManagedAccountSettings {
```

Comment:
I think its sort of a take on Protocol-Based Programming in Swift. We should, ideally, be regenerating those Core Data accessors and not manually adding the properties to the classes. Over time those generated properties will change based upon newer best practices - this forces us to keep those accessors that would normally get overwritten separate. 

I like/hate it too. :smile: